### ISSUE TITLE
Add sitemap, link to in footer

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6250]
WCAG 2.4.5:

> Multiple ways are available to find other web pages on the site - at least two of: a list of
related pages, table of contents, site map, site search, or list of all available web pages.

The quickest way for us to meet this criteria is to add a basic sitemap to PyPI and link to it from the site footer.

**Comment by nlhkabu** [comment]
@woodruffw - I've assigned this one to you as some backend work will be required here.

**Comment by di** [comment]
Sitemap is here: https://pypi.org/sitemap.xml

**Comment by nlhkabu** [comment]
Thanks @di - but I don't know that this would actually be useful/usable for people wanting a secondary method of navigation.

The recommendations say:
> The simplest and most common kind of site map is an outline that shows links to each section or sub-site. Such outline views do not show more complex relationships within the site, such as links between pages in different sections of the site. The site maps for some large sites use headings that expand to show additional detail about each section.

Further guidance on how to create site map:
https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G63

Example site map:
https://www.w3.org/WAI/sitemap/

**Comment by di** [comment]
Sorry, should have added more context! Wanted to point y'all to the existing machine-readable sitemap as the view could be reused for a human-readable sitemap (i.e., we don't need to start from scratch).
